### ISSUE TITLE
Tuple syntax in `IntoSystemSetConfigs::after` and co. not working

### ISSUE THREAD

**Issue by mscofield0** [bevyengine/bevy#10138]
## Bevy version
0.11

## What you did

Try writing `SystemSetA.after((SystemSetB, SystemSetC))`.

## What went wrong

```rust
error[E0277]: the trait bound `(SystemSetB, SystemSetC): IntoSystemSet<_>` is not satisfied
```

The current impls of `IntoSystemSetConfigs` do not support that syntax, although it should.

## Additional information

A workaround is:
```rust
let foo = (SystemSetB, SystemSetC).into();
SystemSetA.after(foo);
```

**Comment by alice-i-cecile** [comment]
Agreed that this syntax should be supported. Fixing this will involve a bit of work wading through the traits, but should be feasible.
